tests/e2e/ansible/install_test_deps.yaml

```diff
@@ -39,7 +39,7 @@
       ignore_errors: yes
     - name: Install kustomize
       shell: |
-        curl -s --retry 3 --retry-delay 10 "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+        curl -s --retry 3 --retry-delay 10 -u ${GITHUB_USER}:${GITHUB_TOKEN} "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
IIRC the GITHUB_USER and GITHUB_TOKEN is not set by default in the CI workflows and one has to specifically set it via env: https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28 or am I wrong? (note curl will happily accept `-u :` so it should work well with empty user as well.

Reply:
@ldoktor locally if I put echo $GITHUB_USER and $GITHUB_TOKEN seems to exists on the TDX machine

Reply:
Well it's not in https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables so might I ask you to add it to the workflows as well to ensure it's going to work properly on all workers?